ui/helpers/utils/util.js

```diff
@@ -631,3 +631,19 @@ export const getNetworkNameFromProviderType = (providerName) => {
 export const isAbleToExportAccount = (keyringType = '') => {
   return !keyringType.includes('Hardware') && !keyringType.includes('Snap');
 };
+
+/**
+ * Groups by key an array of objects
+ *
+ * @param {*} key
+ * @param {*} array
+ * @returns array grouped by a key
+ */
+export const groupbyKey = (key, array) => {
```

Comment:
Do we need this or can we use `lodash` `groupBy`?

Reply:
Definitely! will remove the fct and its tests and replacing it with lodash! 🙏